test/completion.common.vim

```diff
@@ -216,10 +215,13 @@ function! Test_Compl_No_Filetype()
   " mode until done.
   function! Check( id ) closure
     call WaitForCompletion()
+    let items = complete_info().items
+    call map( items, {index, value -> value.word} )
+    call assert_equal( [ 'hello' ], items )
     call feedkeys( "\<ESC>" )
   endfunction
 
-  call FeedAndCheckMain( 'ahe', funcref( 'Check' ) )
+  call FeedAndCheckMain( 'Ahe', funcref( 'Check' ) )
   " Checks run in insert mode, then exit insert mode.
   call assert_false( pumvisible(), 'pumvisible()' )
```

Comment:
Should this be deleted from the test?

Reply:
@puremourning

Reply:
if we got "hello" in completion items then we don't have to check if pum is visible?

Reply:
we need to wait for it, otherwise test might fail/be flaky